drivers/npmx/npmx_shell.c

```diff
@@ -2241,6 +2455,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_npmx, SHELL_CMD(charger, &sub_charger, "Charg
 			       SHELL_CMD(buck, &sub_buck, "Buck", NULL),
 			       SHELL_CMD(ldsw, &sub_ldsw, "LDSW", NULL),
 			       SHELL_CMD(adc, &sub_adc, "ADC", NULL),
+			       SHELL_CMD(timer, &sub_timer, "Timer", NULL),
```

Comment:
This new command should either be at the end of the list or definitions of subcommand should be moved right after subcommand for adc to preserve the ordering.